Document/0x05e-Testing-Cryptography.md

```diff
@@ -233,3 +233,52 @@ public static void main (String args[]) {
 
 ##### Info
 - [#nelenkov] - N. Elenkov, Android Security Internals, No Starch Press, 2014, Chapter 5.
+
+
+### Testing Key Management
+
+#### Overview
+
+Symmetric cryptography provides confidentiality and integrity of data because it ensures one basic cryptographic principle. It is based on the fact that a given ciphertext can only, in any circumstance, be decrypted when providing the original encryption key. The security problem is thereby shifted to securing the key instead of the content that is now securely encrypted. Asymmetric cryptography solves this problem by introducing the concept of a private and public key pair. The public key can be distributed freely, the private key is kept secret.
```

Comment:
True, good catch! That was actually what I meant but got lost during writing it :-)